libs/design/src/atoms/progress-indicator/progress-indicator.component.scss

```diff
@@ -0,0 +1,10 @@
+.daff-progress-indicator {
+    background: rgba(0,0,0,0.1);
+    height: 2px;
+    width:100;
```

Comment:
100...%?

Reply:
resolved in 9c368153db9dd81c88d8806c4cbff5ce74563649